ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -138,12 +135,12 @@ export const AccountListMenu = ({
       distance: 100,
       maxPatternLength: 32,
       minMatchCharLength: 1,
-      keys: ['name', 'address'],
+      keys: ['metadata.name', 'address'],
     });
     fuse.setCollection(accounts);
     searchResults = fuse.search(searchQuery);
   }
-  searchResults = mergeAccounts(searchResults, internalAccounts);
+  searchResults = mergeAccounts(searchResults, accounts);
```

Comment:
Since `accounts` will now have all the data from internalAccounts + the cached balance, we wont need to perform a merge anymore.

Reply:
To be updated in the mergeAccounts pr